eo-runtime/src/test/resources/jul.properties

```diff
@@ -1,2 +1,2 @@
 handlers = org.slf4j.bridge.SLF4JBridgeHandler
-.level=FINEST
\ No newline at end of file
+.level=INFO
```

Comment:
@mximp so you are turning it off?

Reply:
@yegor256 Yes. For the normal builds it's absolutely unnecessary.
It also speeds up the build two times.
It should be enabled for debugging purposes only.

Reply:
@mximp maybe you can add some explanation to README for those who want to see logs? Where to go and what to set up in order to see the logs.

Reply:
That's a good idea.. Done.